src/components/TopicList.vue

```diff
@@ -15,15 +15,16 @@
 </template>
 
 <script>
-import { mapState } from 'vuex';
+import Backend from '../utils/backend';
 import Topic from './tipRecords/Topic.vue';
 import AeAmountFiat from './AeAmountFiat.vue';
 
 export default {
   components: { Topic, AeAmountFiat },
-  computed: mapState({
-    topics: ({ topics }) => topics.filter(([t]) => t !== '#test'),
-  }),
+  data: () => ({ topics: [] }),
+  async mounted() {
+    this.topics = (await Backend.getTopics()).filter(([t]) => t !== '#test');
```

Comment:
This is a tricky thing. I don't think that users would affect this field in most cases, so it should be enough to fetch it once in a session. It is still can be updated by page refreshing. This change simplifies store and rendering in SSR.